src/lib/Scenes/City/Components/Event/index.tsx

```diff
@@ -1,26 +1,26 @@
 import { Box, color, Flex, Sans, Serif } from "@artsy/palette"
-import Button from "lib/Components/Buttons/InvertedButton"
+import { EventMutation } from "__generated__/EventMutation.graphql"
+import InvertedButton from "lib/Components/Buttons/InvertedButton"
 import OpaqueImageView from "lib/Components/OpaqueImageView"
 import SwitchBoard from "lib/NativeModules/SwitchBoard"
 import React from "react"
 import { TouchableWithoutFeedback } from "react-native"
+import { commitMutation, graphql, RelayProp } from "react-relay"
 import styled from "styled-components/native"
 
-const ButtonWrapper = styled(Box)`
-  height: 34;
-  width: 98;
-`
-
 const TextContainer = styled(Box)`
   width: 200;
 `
 
 interface Props {
+  relay: RelayProp
   event: {
     node: {
       name: string
       __id: string
+      _id: string
       id: string
+      is_followed: boolean
```

Comment:
@kierangillen There is no need to type all these typings manually, they map 1:1 to the query made in GlobalMap, right? So you should be able to get that typing like so (as if you were getting a property from a regular JS object):

```ts
import { GlobalMap_viewer } from "__generated__/GlobalMap_viewer.graphql"

interface Props {
  relay: RelayProp
  event: GlobalMap_viewer["city"]["shows"]["edges"][0]
}
```

Alternatively you can make use of a fragment to emit separate typings–and that might be a good step to making all these components actual Relay containers–but that’s not functionally going to be any different than the above. Nonetheless, ping me if you want to pair on that.